lib/pure/os.nim

```diff
@@ -1716,15 +1716,54 @@ proc createSymlink*(src, dest: string) {.noWeirdTarget.} =
     if symlink(src, dest) != 0:
       raiseOSError(osLastError(), $(src, dest))
 
+when defined(windows) and not weirdTarget:
+  proc getFinalPathNameByHandleW(
+    hFile: Handle,
+    lpszFilePath: WideCStringObj,
+    cchFilePath: DWORD,
+    dwFlags: DWORD
```

Comment:
Why 1 argument per line?, is kinda too vertical and long, line is not thaaaat long anyways. :thinking:

Reply:
fixed